google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/models/BackupTest.java

```diff
@@ -17,9 +17,13 @@
 
 import static com.google.common.truth.Truth.assertThat;
 
+import com.google.bigtable.admin.v2.Backup.State;
```

Comment:
Please avoid importing ambiguous inner classes. `Backup.State` is a lot easier to understand then State (same for bigtable Status)

Reply:
I believe that's what we're importing unless I am missing something?

Reply:
I think @igorbernstein2 means the code should read as `Backup.State.READY` as opposted to `State.READY` (see line 91), and same for Status

Reply:
Gotcha, done!